README.md

```diff
@@ -298,6 +298,65 @@ pip install -r requirements.txt
 You can then build the documentation by running ``make <format>`` from the
 ``docs/`` folder. Run ``make`` to get a list of all available output formats.
 
+### Building the CPU-version Libtorch for 32-bit on Windows
+Please use Visual Studio 2017 to the latest version along with installing "VC++ 2017 version 15.4 v14.11 toolset".
+Please use [Anaconda](https://www.anaconda.com/distribution/#download-section) to set 32-bit environment by yourself.
+You need install Python in 32-bit,recommend the 3.7 version.
+Most of steps are same as build pytorch on Windows, a little different with building Libtorch for 32-bit on Windows:
```

Comment:
nit: "Build PyTorch on Windows" (capitalize to match the name); you probably want a conjunction before "a little different"; maybe you want a "but" there?